crates/matrix-sdk-ui/src/notification_client.rs

```diff
@@ -70,6 +76,24 @@ impl NotificationClient {
         NotificationClientBuilder::new(client).await
     }
 
+    /// Fetches the content of a notification using a mix of different
```

Comment:
```suggestion
    /// Fetches the content of a notification.
```

I don't think the “mix of strategies” is necessary for the description of the method.